<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Sales Tex Calculater</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalcBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>99</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Tax</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="TexRate">
       <property name="text">
        <string>TexRate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="TexRate_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Price">
       <property name="text">
        <string>Price</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="Price_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="FinalPrice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="FinalPriceResult">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWhat">
    <property name="title">
     <string>what?</string>
    </property>
   </widget>
   <addaction name="menuWhat"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
